p2p-trend-chef-splunk/azuredeploy.json

```diff
@@ -0,0 +1,1327 @@
+      "metadata": {
+        "description": "Windows Deep Security Agent Virtual Machine size"
+      }
+    },
+    "publicIPDomainNameLabelDSAgentWindows": {
+      "type": "string",
+      "defaultValue": "dnsdsagentwindow",
+      "metadata": {
+        "description": "Windows Deep Security Agent unique public DNS prefix"
+      }
+    },
+    "windowsOSVersion": {
+      "type": "string",
+      "defaultValue": "2012-R2-Datacenter",
+      "allowedValues": [
+        "2008-R2-SP1",
+        "2012-Datacenter",
+        "2012-R2-Datacenter"
+      ],
+      "metadata": {
+        "description": "Windows OS version for Chef Workstation and Windows Deep Security Agents"
+      }
+    },
+    "windowsvmcount": {
+      "type": "int",
+      "defaultValue": 1,
+      "metadata": {
+        "description": "Count of Windows Deep Security Agents to deploy"
+      }
+    },
+    "adminUsernameDSAgent": {
+      "type": "string",
+      "defaultValue": "trend",
+      "metadata": {
+        "description": "Linux Deep Security Agent Virtual Machine admin username"
+      }
+    },
+    "adminPasswordDSAgent": {
+      "type": "securestring",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Linux Deep Security Agent Virtual Machine admin password"
+      }
+    },
+    "vmSizeDSAgent": {
+      "type": "string",
+      "defaultValue": "Standard_D1",
+      "allowedValues": [
+        "Standard_D1"
+      ],
+      "metadata": {
+        "description": "Linux Deep Security Agent Virtual Machine size"
+      }
+    },
+    "publicIPDomainNameLabelDSAgent": {
+      "type": "string",
+      "defaultValue": "dnsdsagentlinux",
+      "metadata": {
+        "description": "Linux Deep Security Agent unique public DNS prefix"
+      }
+    },
+    "ubuntuOSVersion": {
+      "type": "string",
+      "defaultValue": "14.04.2-LTS",
+      "allowedValues": [
+        "12.04.5-LTS",
+        "14.04.2-LTS",
+        "15.10"
+      ],
+      "metadata": {
+        "description": "Linux(Ubuntu) OS version for Linux Deep Security Agents"
+      }
+    },
+    "linuxvmcount": {
+      "type": "int",
+      "defaultValue": 1,
+      "metadata": {
+        "description": "Count of Linux Deep Security Agents to deploy"
+      }
+    },
+    "baseUrl": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URL for dependent deployment files",
+        "artifactsBaseUrl": ""
+      },
+      "defaultValue": "https://trendmicrop2p.blob.core.windows.net/trendmicropushtopilot"
+    },
+    "automationUserName": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Azure account username"
+      }
+    },
+    "automationUserPassword": {
+      "type": "securestring",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Azure account password"
+      }
+    }
+  },
+  "variables": {
+    "location": "[parameters('location')]",
+    "location2": "East US 2",
```

Comment:
We used that for automation (which can be deployed in specific locations). In the changed version we removed that